kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/induction_smelter.js

```diff
@@ -55,6 +55,11 @@ events.listen('recipes', function (event) {
                 inputs: [item.of('minecraft:iron_ingot', 1), ingredient.of('#forge:dusts/coal_coke', 1)],
                 experience: 0.2,
                 outputs: ['emendatusenigmatica:steel_ingot']
+            },
+            {
+                inputs: [item.of('#forge:ingots/iron', 1), ingredient.of('#forge:ingots/lead', 1)],
```

Comment:
`item.of('#forge:ingots/iron', 1)` should be `ingredient.of('#forge:ingots/iron', 1)`
Item.of(tag) -> first item in tag
Ingredient -> The tag

Reply:
I will fix it myself, just letting you know :)

Reply:
Noted for next time, ty